.gitignore

```diff
@@ -61,3 +61,9 @@ lib/qtscript-bytearray/*.cc
 .idea
 *.vscode
 compile_commands.json
+
+# Developers might use `--settingsPath ./devsettings/`
+devsettings
```

Comment:
Is this some standard directory or naming convention. We shouldn't add exclusion rules for arbitrary personal habits.

Reply:
It is the directory that is currently used in the `run` command. We could of course change it both in the command and in the `.gitignore` to something else.

Reply:
Or remove the flag from the `run` command. I just found it weird that the flag is there and, at the same time, that folder is not in the `.gitignore`.